_1_basics/src/main/java/code/_4_student_effort/Main.java

```diff
@@ -3,6 +3,24 @@
 public class Main {
 
   public static void main(String[] args) {
-    //TODO put your code changes in here
+    FizzBuzz(100);
```

Comment:
pentru numarul 105, care este divizibil si cu 3,5,7 cate conditii si if-uri suplimentare ai adauga in plus ?